packages/odyssey-react-mui/src/@types/react-augment.d.ts

```diff
@@ -17,3 +17,7 @@ export interface ForwardRefWithType extends FC<WithForwardRefProps<Option>> {
     FC<WithForwardRefProps<T>>
   >;
 }
+
+export type FocusHandle = {
+  focus: () => void;
+};
```

Comment:
Limits scope of refs that are passed in so they can only hook onto the `.focus()` method